Sources/Tracing/Tracer.swift

```diff
@@ -30,11 +30,15 @@ public protocol Tracer: Instrument {
     ///   - baggage: The `Baggage` providing information on where to start the new ``Span``.
     ///   - kind: The ``SpanKind`` of the new ``Span``.
     ///   - time: The `DispatchTime` at which to start the new ``Span``.
+    ///   - file: The `fileID` where the span was started.
+    ///   - line: The file line where the span was started.
     func startSpan(
         _ operationName: String,
         baggage: Baggage,
         ofKind kind: SpanKind,
-        at time: DispatchWallTime
+        at time: DispatchWallTime,
```

Comment:
why not the new Clock/Duration/... API?

Reply:
Yeah just noticed this too tbh...

I'll need to think about this, it'd force users to require 5.7+ so we may not want that... we'd want vapor to adopt without having to suddenly bump their required swift versions. Probably we can add an public API accepting `Instant` but I guess implementors may have to implement against dispatch time or something else... to keep the swift requirement low :-/ 

I'll think about it more and ticketify. AHC we'd also not want to force to require 5.7 suddenly 🤔

Reply:
We can add an overload though that delegates to the dispatch time one 👍

Reply:
I'll add overload in separate PR

Reply:
https://github.com/apple/swift-distributed-tracing/issues/71